presto-main/src/test/java/io/prestosql/spiller/TestFileSingleStreamSpillerFactory.java

```diff
@@ -48,14 +48,15 @@
 @Test(singleThreaded = true)
 public class TestFileSingleStreamSpillerFactory
 {
-    private final Closer closer = Closer.create();
+    private Closer closer;
     private ListeningExecutorService executor;
     private File spillPath1;
     private File spillPath2;
 
     @BeforeMethod
     public void setUp()
     {
+        closer = Closer.create();
         executor = MoreExecutors.listeningDecorator(Executors.newCachedThreadPool());
```

Comment:
We should call `shutdownNow()` in the cleanup

Reply:
it's called via
https://github.com/prestosql/presto/blob/0bc495b9c3b01929b94b1f4672a834b92a236f2b/presto-main/src/test/java/io/prestosql/spiller/TestFileSingleStreamSpillerFactory.java#L61